doc/api/fs.md

```diff
@@ -1826,6 +1830,8 @@ added: v0.8.6
 
 Synchronous ftruncate(2). Returns `undefined`.
 
+For detailed information, see the documentation of [`fs.ftruncate()`][].
```

Comment:
It seems we miss the bottom reference for this link. You can add this to the bottom list (in ASCII sorting order):
```md
[`fs.ftruncate()`]: #fs_fs_ftruncate_fd_len_callback
```